### ISSUE TITLE
If VCH deletion returns an error, the user is not given any indication that a failure has occurred

### ISSUE THREAD

**Issue by zjs** [vmware/vic-ui#128]
**VIC version:**

`vic-v1.3.0-rc2-2628-9e2a95b6.ova`

**Deployment details:**

vSphere Client version 6.5.0.10000

**Steps to reproduce:**

Delete a VCH in a bad state, or for which deletion will fail for some other reason.

**Actual behavior:**

A 400 is returned by the user, but nothing is shown to the user.

**Expected behavior:**

The fact that an error has occurred is shown the user, along with any error message.

**Logs:**

`vic-machine-server.log`:
```
Nov 28 2017 23:04:18.120Z INFO  op=1.15: Request: DELETE /container/target/sc-rdops-vm16-dhcp-241-196.eng.vmware.com/vch/vm-820 HTTP/1.1
Nov 28 2017 23:04:18.169Z INFO  op=1.15: Validating target
Nov 28 2017 23:04:18.371Z INFO  op=1.15: Response: 400 Bad Request
```

`vsphere_client_virgo.log`:
```
[2017-11-28T23:04:17.660Z] [WARN ] serenity-thread-pool-4       70000128 100006 200001 com.vmware.vise.util.session.SessionUtil                          Just clobbered an existing httpRequest 
[2017-11-28T23:04:17.661Z] [WARN ] serenity-thread-pool-4       70000128 100006 200001 com.vmware.vise.util.session.SessionUtil                          Just clobbered an existing httpSession 
```

**Comment by andrewtchin** [comment]
fix for 1.3, ok if its a mitigation if full fix is risky

**Comment by zjs** [comment]
We should address this for 1.3.0 in some form. If a complete fix is complicated or risky, we should do something simpler or more limited for now, and check the complete fix into master as high priority post-1.3 work.
